locales/en-US/editor.properties

```diff
@@ -126,6 +126,7 @@ CMD_TOGGLE_QUICK_DOCS=Quick Docs
 # Drag and Drop
 
 DND_MAX_FILE_SIZE_EXCEEDED=file exceeds maximum supported size: 3MB
+DND_MAX_ARCHIVE_SIZE_EXCEEDED=file exceeds maximum supported size: 5MB
```

Comment:
Because these 2 strings are basically identical, I think we should try to reuse them, and just pass the number that differs at runtime:

* Let's create a new string, `DND_MAX_SIZE_EXCEEDED=file exceeds maximum supported size: {0} MB.`
* Everywhere that `Strings.DND_MAX_FILE_SIZE_EXCEEDED` is used, change it to `StringUtils.format(Strings.DND_MAX_SIZE_EXCEEDED, "3")`
* Everywhere that `DND_MAX_ARCHIVE_SIZE_EXCEEDED` is used, change it to `StringUtils.format(Strings.DND_MAX_SIZE_EXCEEDED, "5")`

You can see an example of this in:

* https://github.com/MargarytaChepiga/brackets/blob/d2d3cce9f1617137b0c1b9d7e79889c99e51ef24/locales/en-US/editor.properties#L11
* https://github.com/adobe/brackets/blob/f028b3ecdf2017498196f39ddbbe4a2364366c92/src/file/FileUtils.js#L190